pyccel/symbolic/lambdify.py

```diff
@@ -123,7 +123,7 @@ def lambdify(expr, args):
     return func
 
 def set_fst(expr, fst):
-    if isinstance(expr, (tuple,list,Tuple)):
+    if isinstance(expr, (tuple,list)):
```

Comment:
We are still using `Tuple` at line 58 of this file. Can we get rid of it?

Reply:
I believe lambdify uses sympy to create a usable temporary function. But I'm not sure

Reply:
I think it creates a Pyccel `FunctionDef` starting from `sympy.Lambda` or something else...

Reply:
I have just had another look at this. You are right, it should be replaced, but I don't know enough about this file to know if it should be replaced with `tuple` or `PythonTuple`. @saidctb do you know?

Reply:
this is for symbolic manipulations, so leave sympy there for the moment

Reply:
OK then, I think we can merge this PR as it is.